edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/TwinConfigSource.cs

```diff
@@ -199,7 +281,16 @@ enum EventIds
                 ErrorUpdatingLastDesiredStatus,
                 ErrorHandlingDesiredPropertiesUpdate,
                 PatchConfigSuccess,
-                ErrorGettingCachedConfig
+                ErrorGettingCachedConfig,
```

Comment:
Please seek way for code reuse.

Reply:
Events are separate to edge agent and edge hub and right now there is no opportunity to reuse events